h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -214,15 +224,12 @@ class SearchBarController extends Controller {
      */
     const updateHiddenInput = () => {
       let newValue = '';
-      Array.from(this._lozengeContainer.querySelectorAll('.js-lozenge')).forEach((loz) => {
-
-        let inputValue = loz.querySelector('.js-lozenge__content').textContent;
-
+      lozenges().forEach((loz) => {
+        let inputValue = loz.inputValue();
```

Comment:
Oh, nice, so `LozengeController` is now responsible for saying what contribution its lozenge should make to the search input. Nice